strawberry_django/type.py

```diff
@@ -79,9 +80,29 @@ def _process_type(
     select_related: Optional[TypeOrSequence[str]] = None,
     prefetch_related: Optional[TypeOrSequence[PrefetchType]] = None,
     disable_optimization: bool = False,
+    fields: Optional[Union[List[str], str]] = None,
+    exclude: Optional[List[str]] = None,
     **kwargs,
 ) -> _O:
     is_input = kwargs.get("is_input", False)
+
+    if not exclude:
+        exclude = []
+
+    if fields == "__all__":
+        model_fields = list(model._meta.fields)
+    elif isinstance(fields, collections.abc.Sequence):
```

Comment:
Since you are checking here for `Sequence`, you should type it as `Sequence` as well in the arguments

Also, you should be importing `Sequence` from `typing` to avoid the issue pyright is raising

Reply:
Also, I would type this as a `Collection`, since you are not actually using this for iteration but instead to check if a name is "contained" in that